src/compas_fab/robots/robot.py

```diff
@@ -939,16 +1111,34 @@ def constraints_from_configuration(self, configuration, tolerances, group=None):
 
         Parameters
         ----------
-        configuration: :class:`compas_fab.robots.Configuration`
+        configuration: :class:`Configuration`
             The target configuration.
         tolerances: list of float
-            The tolerances (as +/-) on each of the joints defining the bound in radian
-            to be achieved. If only one value is passed it will be used to create
-            bounds for all joint constraints.
+            The tolerances (as positive and negative values) on each of the
```

Comment:
Both the existing and suggested options are hard to understand. The tolerances list is one float value per joint, where that value defined the lower/upper bound such that `[configuration.values[j] - tolerance, configuration.values[j] + tolerance]`

Reply:
d421722